charts/k8s-monitoring/templates/agent_config/_traces.river.txt

```diff
@@ -1,16 +1,26 @@
 {{ define "agent.config.traces" }}
-{{- include "checkForTracePort" . -}}
-{{- with .Values.traces }}
 // Traces
 otelcol.receiver.otlp "trace_receiver" {
+{{- if .Values.traces.receiver.grpc.enabled }}
```

Comment:
The older .receiver.port was kept for compatibility but looks like it was dropped from the template iiuc - do you need another conditional to check for that?

Reply:
Yes, thank you!